src/Media.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+
+export interface MediaProps {
+  query: string,
+  children: (match: boolean | undefined) => React.ReactNode;
+}
+
+export class Media extends React.Component<MediaProps, {
```

Comment:
Home made React Media. Just a few bytes, while React-Media is 30kb due to json2mq.